tools/set-metadata.ts

```diff
@@ -18,7 +27,9 @@ async function main() {
     // set all the packages peerDependencies to be the same as root package.json version
     for (const p of packages) {
```

Comment:
can immediately rename p to package?

Reply:
`package` is a reserved word that is not valid in strict mode. I could call it something else, but this isn't code that I deliberately changed in this PR so I am not going to worry about this.
Prettier got excited when I opened the file and reformatted the lines! 🤕

Reply:
rename to packageName?)

Reply:
Ok, I have renamed it. But this is unrelated to my PR!
As I said, prettier made the line format change.

Reply:
`packageInfo` reflects a better name for this btw